<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:abcast/1 2024-09-19 15:49:01.218 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:49:01.218531 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: &lt;0.244171.0&gt;
    registered_name: my_test_name
    exception exit: {&lt;0.244169.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [&lt;0.244169.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3457
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-09-19 15:49:03.318 ***</b>
=ERROR REPORT==== 19-Sep-2024::15:49:03.318418 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244197.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-09-19 15:49:03.319 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:49:03.318948 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244202.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244197.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244197.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-09-19 15:49:43.245 ***</b>
=ERROR REPORT==== 19-Sep-2024::15:49:43.245734 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248547.0&gt;,
                               {#Ref&lt;0.1014354164.697040897.168143&gt;,1008057,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.1014354164.697040897.168143&gt;,1008057,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-09-19 15:49:43.246 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:49:43.246101 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248567.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1014354164.697040897.168143&gt;,1008057,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248547.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4504
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init_local_registered_parent/1 2024-09-19 15:49:43.269 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:49:43.269083 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_local/2
    pid: &lt;0.248573.0&gt;
    registered_name: my_server
    exception exit: {&lt;0.248571.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [foobar]
    message_queue_len: 0
    messages: []
    links: [&lt;0.248571.0&gt;]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3439
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-09-19 15:54:17.963 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:54:17.963433 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254333.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.1014354164.698613761.82022&gt;,6353,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254331.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 28661
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-19 15:54:27.478 ***</b>
=ERROR REPORT==== 19-Sep-2024::15:54:27.478225 ===
** Generic server <0.255606.0> terminating 
** Last message in was {nodedown,'peer_down_crash-435748-11810@9b54c569654f'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-435748-11810",
                              peer_down => crash},
                            'peer_down_crash-435748-11810@9b54c569654f',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-435748-11810",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDliNTRjNTY5NjU0ZgAD5nYAAAAAZuwpfA=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-435748-11810@9b54c569654f'}},
                            {<0.255595.0>,#Ref<0.1014354164.691011587.99835>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-435748-11810@9b54c569654f'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-09-19 15:54:27.479 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:54:27.479041 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255606.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-435748-11810@9b54c569654f'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.255595.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11140
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-19 15:54:28.402 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:27.401181 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-445443-11810@9b54c569654f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-19 15:54:28.471 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:27.469310 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_halt-445347-11810@9b54c569654f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-19 15:54:28.804 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:28.804384 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-501890-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-09-19 15:54:28.990 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:28.990268 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-445379-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-09-19 15:54:44.512 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:43.510528 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-371493-11810@9b54c569654f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-09-19 15:54:44.967 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:44.967389 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-445539-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-09-19 15:54:45.075 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:45.075383 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-371461-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:47.151 ***</b>
=ERROR REPORT==== 19-Sep-2024::15:54:47.151189 ===
** Generic server <0.255982.0> terminating 
** Last message in was {'EXIT',<0.255954.0>,
                               {#Ref<0.1014354164.690749444.62154>,1550586,ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-436580-11810",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-436580-11810@9b54c569654f',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-436580-11810",
                                     "-user","peer"],
                                    #Port<0.3118>,undefined,<<>>,running,
                                    {<0.255954.0>,
                                     #Ref<0.1014354164.691011587.103399>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1014354164.690749444.62154>,1550586,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:47.151 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:54:47.151741 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255982.0>
    registered_name: []
    exception exit: {#Ref<0.1014354164.690749444.62154>,1550586,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.255954.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3118>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 13488
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:47.716 ***</b>
=ERROR REPORT==== 19-Sep-2024::15:54:47.716315 ===
** Generic server <0.256009.0> terminating 
** Last message in was {'EXIT',<0.255950.0>,
                           {#Ref<0.1014354164.690749444.62061>,2117052,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "KPEJJLKGLHZNQKIHAVWM","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@9b54c569654f.2024-09-19_15.39.07/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-436708-11810",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@9b54c569654f.2024-09-19_15.39.07/make_test_dir.stdlib_test.logs/run.2024-09-19_15.39.44/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-436708-11810",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.3/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-436708-11810@9b54c569654f',
                                    "/usr/local/lib/erlang-26.2.5.3/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-436708-11810",
                                     "-setcookie","KPEJJLKGLHZNQKIHAVWM",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@9b54c569654f.2024-09-19_15.39.07/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-436708-11810",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@9b54c569654f.2024-09-19_15.39.07/make_test_dir.stdlib_test.logs/run.2024-09-19_15.39.44/log_private/",
                                     "-user","peer"],
                                    #Port<0.3145>,undefined,<<>>,running,
                                    {<0.255950.0>,
                                     #Ref<0.1014354164.698613761.99432>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.1014354164.690749444.62061>,2117052,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:47.717 ***</b>
=CRASH REPORT==== 19-Sep-2024::15:54:47.717172 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256009.0>
    registered_name: []
    exception exit: {#Ref<0.1014354164.690749444.62061>,2117052,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.255950.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3145>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 10958
    stack_size: 29
    reductions: 20861
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:48.124 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:47.122724 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-436676-11810@9b54c569654f' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:48.133 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:48.133280 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-372197-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:48.561 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:48.561363 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-436548-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-09-19 15:54:48.568 ***</b>
=WARNING REPORT==== 19-Sep-2024::15:54:48.568217 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-372229-11810@9b54c569654f'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-19 16:05:40.710 ***</b>
=ERROR REPORT==== 19-Sep-2024::16:05:40.710744 ===
** Generic server <0.302716.0> terminating 
** Last message in was {'EXIT',<0.302713.0>,
                               {#Ref<0.1014354164.735051777.240938>,350791,
                                [{child1,<0.302722.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302716.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302722.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459159],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1014354164.735051777.240938>,350791,
    [{child1,<0.302722.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-19 16:05:40.711 ***</b>
=ERROR REPORT==== 19-Sep-2024::16:05:40.710748 ===
** Generic server <0.302715.0> terminating 
** Last message in was {'EXIT',<0.302713.0>,
                               {#Ref<0.1014354164.735051777.240938>,350791,
                                [{child1,<0.302722.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302715.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302721.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459159],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.1014354164.735051777.240938>,350791,
    [{child1,<0.302722.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-19 16:05:40.711 ***</b>
=CRASH REPORT==== 19-Sep-2024::16:05:40.711310 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302716.0>
    registered_name: []
    exception exit: {#Ref<0.1014354164.735051777.240938>,350791,
                     [{child1,<0.302722.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.302713.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14905
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-09-19 16:05:40.711 ***</b>
=CRASH REPORT==== 19-Sep-2024::16:05:40.711588 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302715.0>
    registered_name: []
    exception exit: {#Ref<0.1014354164.735051777.240938>,350791,
                     [{child1,<0.302722.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.302713.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 14992
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-09-19 16:06:24.123 ***</b>
=CRASH REPORT==== 19-Sep-2024::16:06:24.122901 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.402873.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [bridge9212,<0.402868.0>]
    message_queue_len: 0
    messages: []
    links: [<0.402874.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4062
  neighbours:
    neighbour:
      pid: <0.402874.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.402873.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Thu Sep 19 2024 15:39:44<!--/date--><br />
</div>
</center>
</body>
</html>
